.github/workflows/release-docker-image-demo.yml

```diff
@@ -0,0 +1,81 @@
+name: Release process
+
+on: push
+
+env:
+  LONG_COMMIT_HASH: 058650a33d36df49cde582cb480666c080af3d64
+  SHORT_COMMIT_HASH: 058650a
+  SEMVER_VERSION: 0.0.1
+
+jobs:
+  create-release:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{env.LONG_COMMIT_HASH}}
```

Comment:
Not sure if you really want to control release with a *static* pipeline, `tags` would be more appropriate.